app/src/main/java/hacking/to/the/gate/Effect.java

```diff
@@ -0,0 +1,8 @@
+package hacking.to.the.gate;
+
+/**
+ * Created by Ruiqian on 9/6/2015.
+ */
+public interface Effect {
+    void effectOfKillingEnemy(int posX);
+}
```

Comment:
When you try to name a class or method, try to describe WHAT IT IS, not HOW DOES IT WORK. In this case, a better name would be.
public interface JetLifeCycle{
    void onDeath();
}

The reason why we do this, is because the implementation of the interface, which is How it works, can always change. But the purpose of the callback, which is What it is, is not likely to change.